src/styles/blocks/nav.scss

```
@@ -19,6 +19,7 @@
   &__link {
     text-decoration: none;
     color: $blackColor;
+    transition: 0.4s;
```

Always list all styles that you apply transition for.

```suggestion
    transition: color 0.4s;
```